mini_pupper_navigation/launch/bringup.launch.py

```diff
@@ -63,6 +63,9 @@ def generate_launch_description():
     nav_configuration_basename = LaunchConfiguration(
         'nav_configuration_basename')
     load_state_filename = LaunchConfiguration('load_state_filename')
+    imu_enable = LaunchConfiguration('imu_enable')
```

Comment:
```suggestion
    use_imu = LaunchConfiguration('use_imu')
```

How about aligning it with the others and use_xxx?